src/main/java/calculator/util/Validator.java

```diff
@@ -0,0 +1,18 @@
+package calculator.util;
+
+import java.util.Objects;
+
+public class Validator {
+
+    public static void validateInput(String input) {
+        if(Objects.isNull(input))
```

Comment:
컨벤션 한번 확인해주시면 좋을 것 같아요 😄 
```suggestion
        if (Objects.isNull(input))
```